python/singa/sonnx.py

```diff
@@ -0,0 +1,242 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+from __future__ import division
+import pickle
+from singa import tensor
+from singa.tensor import Tensor
+from singa import autograd
+from singa import optimizer
+import numpy as np
+import onnx
+from onnx import numpy_helper
+
+from collections import Counter, deque
+import numpy as np
+import math
+
+from .tensor import Tensor
+from . import layer
+from singa.proto import model_pb2
+from . import singa_wrap as singa
+#from .tensor import einsum
+from autograd import *
+
+def load_onnx_model(name = 'singonnx.pkl'):
+    '''
+        load onnx model
+        Args:
+            name:name of onnx model file
+        Return:
+            onnx model
+    '''
+    with open(name, 'rb') as f:
+        model = pickle.load(f)
+    return model
+
+def onnx_model_init(inputs,model):
```

Comment:
if can merge the function 'load_onnx_model' and 'onnx_model_init'? the first function seems unnecessary.